src/meetingreadinesschecker/DefaultMeetingReadinessChecker.ts

```diff
@@ -44,7 +44,9 @@ export default class DefaultMeetingReadinessChecker implements MeetingReadinessC
     private configuration: MeetingReadinessCheckerConfiguration = new MeetingReadinessCheckerConfiguration()
   ) {}
 
-  async checkAudioInput(audioInputDeviceInfo: MediaDeviceInfo): Promise<CheckAudioInputFeedback> {
+  async checkAudioInput(
+    audioInputDeviceInfo: MediaDeviceInfo | string
+  ): Promise<CheckAudioInputFeedback> {
```

Comment:
Just change it to

```
async checkAudioInput(input: Device)
```

Any caller who was providing a `MediaDeviceInfo` was actually providing a `MediaTrackConstraints` instance, so let's just be accurate about it.

Reply:
I actually used Device first then realized that Device also contains MediaStream which doesn't work with all check here (like the Camera one) so I just do string instead.

Reply:
Why would we not allow testing a stream?

Reply:
Update to Device